docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+services:
+  nginx:
+    image: nginx:latest
+    container_name: nginx
+    ports:
+      - "80:80"
+    volumes:
+      - ./nginx/nginx.conf:/etc/nginx/conf.d/nginx.conf
+      - ../rest-api:/home/restapi/http
+    networks:
+      - app-network
+
+  php:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: php
+    volumes:
+      - ../rest-api:/home/restapi/http
+    networks:
+      - app-network
+
+  db:
+    container_name: mysql
+    image: mysql:5.7
+    ports:
+      - "3306:3306"
```

Comment:
You can have more thab 1 docker running, so on this case I prefer configurable ports. Just use here env variable. `${DOCKER_MYSQL_PORT}:3306`